conf/upgrade.yaml

```diff
@@ -0,0 +1,53 @@
+UPGRADE:
+  FROM_VERSION: "6.8"
+  TO_VERSION: "6.9"
+  OS: "rhel7"
+  DISTRO: ""
+  CAPSULE_AK:
+    CAPSULE_AK_RHEL6: "capsule_rhel6"
+    CAPSULE_AK_RHEL7: "capsule_rhel7"
+    CAPSULE_AK_RHEL8: "capsule_rhel8"
+  CLIENT_AK:
+    CLIENT_AK_RHEL6: "clientak_rhel6"
+    CLIENT_AK_RHEL7: "clientak_rhel7"
+    CLIENT_AK_RHEL8: "clientak_rhel8"
+  USER_DEFINED_CAPSULE_AK: ""
+  BASE_URL: ""  # Satellite6 repos url
+  CAPSULE_URL: ""  # Capsule repos
+  PRODUCTS:
+    - "satellite"
+    - "capsule"
+    - "client"
+    - "longrun"
+    - "n-1"
+  DISTRIBUTION: "downstream"
+  FOREMAN_MAINTAIN_SATELLITE_UPGRADE: "true"
+  SATELLITE_HOSTNAME: ""
+  CAPSULE_HOSTNAME: ""
+  DOWNSTREAM_FM_UPGRADE: "false"
+  WHITELIST_PARAM: ""
+  FOREMAN_MAINTAIN_CAPSULE_UPGRADE: "false"
+  USER_DEFINED_CLIENT_HOSTS:
+    CLIENT6_HOSTS: ""
+    CLIENT7_HOSTS: ""
+    CLIENT8_HOSTS: ""
+  CLIENTS_COUNT: "8"
+  EXISTENCE_TEST:
+    REMOTE_EXECUTION_SSH_PASSWORD: ""
+    OAUTH_CONSUMER_KEY: ""
+    OAUTH_CONSUMER_SECRET: ""
+    ENDPOINT: ""
+    SUPPORTED_SAT_VERSIONS:
+      - "6.2"
```

Comment:
Satellite 6.2, 6.3, 6.4, and 6.5 are not supported anymore. See https://access.redhat.com/support/policy/updates/satellite
Probably we should only keep 6.5 onwads versions(6.5 included). what you think?

Reply:
fixed.

Reply:
fixed.